sdk/include/psdk/windowsx.h

```diff
@@ -509,7 +509,7 @@
 #define Static_SetIcon(hwndCtl,hIcon) ((HICON)(UINT)(DWORD)SendMessage((hwndCtl),STM_SETICON,(WPARAM)(HICON)(hIcon),0))
 #define Static_SetText(hwndCtl,lpsz) SetWindowText((hwndCtl),(lpsz))
 #define SubclassDialog(hwndDlg,lpfn) ((DLGPROC)SetWindowLongPtr(hwndDlg,DWLP_DLGPROC,(LPARAM)(DLGPROC)(lpfn)))
-#define SubclassWindow(hwnd,lpfn) ((WNDPROC)SetWindowLongPtr((hwnd),GWLP_WNDPROC,(LPARAM)(WNDPROC)(lpfn)))
+//#define SubclassWindow(hwnd,lpfn) ((WNDPROC)SetWindowLongPtr((hwnd),GWLP_WNDPROC,(LPARAM)(WNDPROC)(lpfn)))
```

Comment:
this macro has conflict and used nowhere. May be delete it?

Reply:
No, it should not be removed.

Reply:
Has conflicts? With which other definitions?

Reply:
Yes, building failed with message "macro SubclassWindow requires 2 arguments, but only 1 given".
SubclassWindow function have one argument.

Reply:
> SubclassWindow function have one argument.

Because you are seeing the override from atlwin.h .
See the last results in: https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=SubclassWindow
The correct way to proceed would be to include the windowsx.h before atlwin.h & ATL consorts.

Reply:
I don't think that makes a difference, ATL does a push_macro/pop_macro on it. Just `#undef SubclassWindow` after including windowsx.h.

Reply:
Ok, i add #undef SubclassWindow